app-modules/portal/src/Http/Controllers/KnowledgeManagement/KnowledgeManagementPortalLogoutController.php

```diff
@@ -46,11 +46,15 @@ class KnowledgeManagementPortalLogoutController extends Controller
     public function __invoke(Request $request)
     {
         /** @var Student|Prospect $user */
-        $user = auth('student')->user() ?? auth('prospect')->user();
+        $user = auth('sanctum')->user();
```

Comment:
We also won't want to do this, as this would enable `users` to login to the portal - they should not be able to.